README-install.md

```diff
@@ -141,14 +141,14 @@ and which parameters you may want to customize, depending on your environment.
 1. Download the files [vars.yaml.template](vars.yaml.template) and
 [ansible-inventory-origin-36-aio](ansible-inventory-origin-36-aio)
 
-    # curl https://raw.githubusercontent.com/ViaQ/Main/master/vars.yaml.template > vars.yaml.template
-    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-36-aio > ansible-inventory
+        # curl https://raw.githubusercontent.com/ViaQ/Main/master/vars.yaml.template > vars.yaml.template
+        # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-36-aio > ansible-inventory
 
 To use ViaQ on Red Hat OCP, use the
 [ansible-inventory-ocp-36-aio](ansible-inventory-ocp-36-aio) file instead
 of the origin-36-aio file (you still need vars.yaml.template):
 
-    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-ocp-36-aio > ansible-inventory
+        # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-ocp-36-aio > ansible-inventory
```

Comment:
This does not need the extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#customizing-varsyaml